.github/workflows/before-surge.sh

```diff
@@ -8,7 +8,6 @@ mkdir .static/components
 mkdir .static/containers
 mkdir .static/dataviz
 mkdir .static/design-system
-mkdir .static/design-tokens
```

Comment:
current design-tokens sb is empty :

![image](https://user-images.githubusercontent.com/19857479/201881702-0ab2b18c-6e5a-4584-9081-b879ed6eb971.png)